### PR TITLE
[MIG] operating_unit to v10.0

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Operating Unit
==============

An operating unit (OU) is an organizational entity part of a company, with
separate management ownership. Management by OU is aimed to introduce the
following features:

* Data partitioning from other OU.
* Own sequencing schemes.
* User access administration for data processing and reporting.
* Product and customer independence of OU.
* OU specific P&L and Balance sheet.

### CODE REVIEW

File: operating_unit/README.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
-   :target: https://www.gnu.org/licenses/lgpl.html
-   :alt: License: LGPL-3
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (JordiBForgeFlow) [inline operating_unit/README.rst]: License should be LGPL

File: operating_unit/README.rst [hunk 2]
```diff
@@ -39,6 +39,14 @@ Other modules extend the standard Odoo apps with the OU.
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/213/9.0
 
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (JordiBForgeFlow) [inline operating_unit/README.rst]: Remove that. It is only used to help newcomers

File: operating_unit/README.rst [hunk 3]
```diff
@@ -39,6 +39,14 @@ Other modules extend the standard Odoo apps with the OU.
...
+.. branch is "10.0" for example
+
+Known issues / Roadmap
```
Comment (JordiBForgeFlow) [inline operating_unit/README.rst]: Remove unused sections

File: operating_unit/__manifest__.py [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+# © 2015-2017 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
 {
     "name": "Operating Unit",
     "summary": "An operating unit (OU) is an organizational entity part of a "
                "company",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
     "author": "Eficent Business and IT Consulting Services S.L., "
```
Comment (JordiBForgeFlow) [inline operating_unit/__manifest__.py]: Rename Eficent Business and IT Consulting Services S.L. to just Eficent

**Comment by JordiBForgeFlow** [comment]
I am missing the place where you change in __manifest__.py the 'installable': True

File: operating_unit/demo/operating_unit_demo.xml [hunk 5]
```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data noupdate="0">
-
+<odoo>
```
Comment (LoisRForgeFlow) [inline operating_unit/demo/operating_unit_demo.xml]: copy the noupdate="0" to the renamed \<odoo\> tag.

File: operating_unit/data/operating_unit_data.xml [hunk 6]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
     <data noupdate="1">
```
Comment (LoisRForgeFlow) [inline operating_unit/data/operating_unit_data.xml]: copy the noupdate="1" to the renamed <odoo> tag.

File: operating_unit/security/operating_unit_security.xml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
 <data noupdate="1">
```
Comment (LoisRForgeFlow) [inline operating_unit/security/operating_unit_security.xml]: copy the noupdate="1" to the renamed \<odoo\> tag. Remove \<data\> tag.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Check tags.

**Comment by LoisRForgeFlow** [comment]
Travis seems to be stuck. I don't know the reason.

**Review by AaronHForgeFlow — COMMENTED** [review]
IMHO The group Administration / Access Right should be Operating unit user and the group Administration / Settings should be able to be manage Operating units

**Comment by LoisRForgeFlow** [comment]
Agree with @aheficent comment about rights.

**Comment by AaronHForgeFlow** [comment]
Why travis is not starting? Is because transifex is missing in travis config file?

**Comment by JordiBForgeFlow** [comment]
@moylop260 Can you help us here? Travis fails

**Comment by LoisRForgeFlow** [comment]
Runbot is running correctly now. 

I agree with @aheficent changes but if you want to leave it as it is this PR is ready to merge.

cc @jbeficent @guewen

**Review by AaronHForgeFlow — APPROVED** [review]
Runbot is working perfectly :+1: 

Regarding my previous comment, in case of changes those have to be done in another PR. The security settings have to be the same as they are in v9

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**